src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -308,8 +308,11 @@ private NuGetVersion GetStandardOrExtendedBundleVersion(IList<NuGetVersion> orde
         {
             if (orderedByDescBundlesList.Count > 1)
             {
+                var previous = orderedByDescBundlesList[1];
+                _logger.LogInformation("Applying platform release channel configuration {platformReleaseChannelName}. Bundle version {previous} will be used", _platformReleaseChannel, previous);
```

Comment:
Should we print the latest one here too? 
something like using bundle version x instead of y.

Reply:
Maybe:

`Applying platform release channel configuration {platformReleaseChannelName}. Previous bundle version {previous} will be used instead of latest version {latest}.`

Reply:
That looks good to me.